tests/auth_tests/test_validators.py

```diff
@@ -127,6 +129,26 @@ def test_validate(self):
             UserAttributeSimilarityValidator(user_attributes=['first_name']).validate('testclient', user=user)
         )
 
+    def test_validate_property(self):
+        class TestUser(Model):
+            email = EmailField()
```

Comment:
Can we remove the `email` field and the tests around it? It looks to me like that's not important for the issue that's solved.